FirebaseInstallations/Source/Library/FIRInstallations.m

```diff
@@ -153,21 +153,20 @@ + (void)validateAppOptions:(FIROptions *)appOptions appName:(NSString *)appName
 
 #pragma mark - Public
 
-+ (FIRInstallations *)installations {
-  FIRApp *defaultApp = [FIRApp defaultApp];
++ (nullable FIRInstallations *)installations {
+  FIRApp *defaultApp = [FIRApp defaultApp];  // Missing configure will be logged here.
   if (!defaultApp) {
-    [NSException raise:kFirebaseInstallationsErrorDomain
-                format:@"The default FirebaseApp instance must be configured before the default"
-                       @"FirebaseApp instance can be initialized. One way to ensure that is to "
-                       @"call `[FIRApp configure];` (`FirebaseApp.configure()` in Swift) in the App"
-                       @" Delegate's `application:didFinishLaunchingWithOptions:` "
-                       @"(`application(_:didFinishLaunchingWithOptions:)` in Swift)."];
+    return nil;
```

Comment:
The old exception text was quite helpful. Consider logging a warning with that text in it.